### PR TITLE
Updating roadmap image to latest

### PR DESCRIPTION
**Pull request by anawishnoff**
## Description
Removed the outdated roadmap image for WinUI 3 and updated it to show the latest one featured in the last community call.

Thanks to @dotMorten for bringing this to my attention.

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -121,5 +121,5 @@ You can download WinUI 3 Preview 1 to try out - we'd love feedback on it, althou
 
 See the image below for a more detailed look at our plans for getting WinUI 3.0 to market: 
 
-![Winui roadmap](images/winui-roadmap-img.PNG)
+![Winui roadmap](WinUI-RoadmapJuly2020.png)
```
Comment (StephenLPeters) [inline docs/roadmap.md:124]: >RoadmapJuly2020 [](start = 23, length = 15)

should this be in the images directory?

Comment (anawishnoff) [reply docs/roadmap.md:124]: Whoops, yes, moving it in upcoming commit

**Comment by anawishnoff** [comment]
@StephenLPeters Also added updates to P1->P2 upgrade instructions in this PR.

**Comment by StephenLPeters** [comment]
you should be able to kick off the pipeline yourself if you'd like by adding a comment with /azp run

**Comment by StephenLPeters** [comment]
Looks good :)

**Comment by anawishnoff** [comment]
@StephenLPeters Thanks! I'll do that from now on.

**Comment by StephenLPeters** [comment]
@anawishnoff can you merge this with master to pick up the fix for the pipeline failure we are seeing here?

**Comment by anawishnoff** [comment]
Hey @StephenLPeters not sure why these tests are failing. I merged with master again yesterday. Any ideas?

**Comment by Felix-Dev** [comment]
@anawishnoff Looking at the failing test - TabViewTests.GamePadTest - pretty sure you just got "unlucky" here and there's a good chance another run of the test pipeline will succeed. Sometimes it just happens that completely unrelated tests fail.

**Comment by dotMorten** [comment]
I'm sure it'll work if you change the roadmap so WinUI3 will be release this year ;-)

**Pull request merged**